dependencies.yml

```diff
@@ -0,0 +1,6 @@
+version: 2
+dependencies:
+- type: php
+  path: /
+  settings:
+    composer_options: "--ignore-platform-reqs"
```

Comment:
Why is this needed? The repository does not use platform requirements.

Reply:
Nonsense. It does use platform requirements, but they are actually needed here.

Reply:
I've copied this [from the docs](https://www.dependencies.io/docs/dependencies/php/).

I think `--ignore-platform-reqs` is relevant because we don't have control over the container used by the service. Not entirely sure though. 

In any case, if Dependencies.io generates an invalid `composer.lock`, Travis will catch it and the build will fail.

Reply:
I think we need to remove this, as we need to lock the platform to PHP 5.4 for the Phar. Otherwise, we'll build a PHP 7.2-only Phar.

Reply:
I think the biggest reason `--ignore-platform-reqs` comes into play is the PHP version. We currently only [ship the PHP container in PHP 7](https://github.com/dependencies-io/php/blob/master/Dockerfile). Does that help clarify anything for you?

Reply:
Related: https://github.com/dependencies-io/php/issues/24

Reply:
@schlessera I don't think it will have an impact on the Phar building.

Dependencies.io is functionally equivalent to running `composer update` locally. If you're already running that locally on PHP 7, we don't have any issues, right?

Reply:
Yeah, I think that's a good way to look at it. If you don't want to commit this config yet, in the Dependencies.io interface you can try clicking "Update composer.lock" to have it try opening a PR using the default PHP settings.

Reply:
My bad, I was thinking about "ignoring the `composer.json` "platform" config", but this is "ignoring the container's PHP runtime version that executes Composer" instead.